include/mrdox/Platform.hpp

```diff
@@ -63,6 +64,16 @@ namespace mrdox {
 # error unknown platform for dynamic linking
 #endif
 
+//------------------------------------------------
+
+#define MRDOX_ASSERT(x) assert(x)
```

Comment:
`#ifndef MRDOX_ASSERT` first?

Reply:
quoth The Falco "this is not a boost library". it's already inside the header guard anyways

Reply:
but someone might want to change it

Reply:
very well